src/Entity/ElectedRepresentative/Mandate.php

```diff
@@ -49,9 +50,20 @@ class Mandate
      *     "value !== null or (value == null and this.getType() === constant('App\\Entity\\ElectedRepresentative\\MandateTypeEnum::EURO_DEPUTY'))",
      *     message="Le périmètre géographique est obligatoire."
      * )
+     *
+     * @deprecated Will be replace by $geoZone
      */
     private $zone;
 
+    /**
+     * @var GeoZone|null
+     *
+     * @ORM\ManyToOne(targetEntity="App\Entity\Geo\Zone")
+     *
+     * @Assert\NotBlank
```

Comment:
it will block updates on admin?

Reply:
Good question, I don't know how the admin form is configured.
If the form gathers all fields automatically, yes we have a problem.
If this new field isn't mapped, I think it may pass.

To prevent any problems I'll remove this assert, and place this in the subsequent PRs

Reply:
done via 927a1f351d716effd5646ae478fe3f2d9b0789ab